results-processor/main.py

```diff
@@ -58,8 +60,8 @@ def decorated_func(*args, **kwargs):
                 return func(*args, **kwargs)
         except filelock.Timeout:
             app.logger.info('%s unable to acquire lock.', func.__name__)
-            # 503 Service Unavailable
-            return 'A result is currently being processed.', 503
+            return ('A result is currently being processed.',
```

Comment:
Will this cause app engine to spin up another instance if persistent?

Reply:
No, I haven't enabled auto scaling for the processor, but it could. See #187 for more details.